tests/terraform/createcluster/createcluster_test.go

```diff
@@ -26,84 +23,101 @@ func Test_TFClusterCreateValidation(t *testing.T) {
 }
 
 var _ = Describe("Test:", func() {
+
 	Context("Build Cluster:", func() {
 		It("Starts up with no issues", func() {
-			status, err := terraform.BuildCluster(&testing.T{}, *tfVars, false)
+			status, err := BuildCluster(&testing.T{}, false)
 			Expect(err).NotTo(HaveOccurred())
 			Expect(status).To(Equal("cluster created"))
+
 			defer GinkgoRecover()
-			fmt.Println("Server Node IPS:", terraform.MasterIPs)
-			fmt.Println("Agent Node IPS:", terraform.WorkerIPs)
-			terraform.PrintFileContents(terraform.KubeConfigFile)
-			Expect(terraform.MasterIPs).ShouldNot(BeEmpty())
-			if terraform.NumWorkers > 0 {
-				Expect(terraform.WorkerIPs).ShouldNot(BeEmpty())
+
+			fmt.Println("Server Node IPS:", MasterIPs)
+			fmt.Println("Agent Node IPS:", WorkerIPs)
+			fmt.Println(KubeConfigFile)
+
+			Expect(MasterIPs).ShouldNot(BeEmpty())
+			if NumWorkers > 0 {
+				Expect(WorkerIPs).ShouldNot(BeEmpty())
 			} else {
-				Expect(terraform.WorkerIPs).Should(BeEmpty())
+				Expect(WorkerIPs).Should(BeEmpty())
 			}
-			Expect(terraform.KubeConfigFile).ShouldNot(BeEmpty())
+			Expect(KubeConfigFile).ShouldNot(BeEmpty())
 		})
 
 		It("Checks Node and Pod Status", func() {
 			defer func() {
-				_, err := terraform.Nodes(terraform.KubeConfigFile, true)
+				fmt.Printf("\nFetching node status\n")
+
+				_, err := terraform.Nodes(KubeConfigFile, true)
 				if err != nil {
 					fmt.Println("Error retrieving nodes: ", err)
 				}
-				_, err = terraform.Pods(terraform.KubeConfigFile, true)
+				_, err = terraform.Pods(KubeConfigFile, true)
 				if err != nil {
 					fmt.Println("Error retrieving pods: ", err)
 				}
 			}()
 
-			fmt.Printf("\nFetching node status\n")
-			expectedNodeCount := terraform.NumServers + terraform.NumWorkers
+			expectedNodeCount := NumServers + NumWorkers
 			Eventually(func(g Gomega) {
-				nodes, err := terraform.Nodes(terraform.KubeConfigFile, false)
+				nodes, err := terraform.Nodes(KubeConfigFile, false)
+
 				g.Expect(err).NotTo(HaveOccurred())
-				g.Expect(len(nodes)).To(Equal(expectedNodeCount), "Number of nodes should match the spec")
+				g.Expect(len(nodes)).To(Equal(expectedNodeCount),
+					"Number of nodes should match the spec")
+
 				for _, node := range nodes {
-					g.Expect(node.Status).Should(Equal("Ready"), "Nodes should all be in Ready state")
+					g.Expect(node.Status).Should(Equal("Ready"),
+						"Nodes should all be in Ready state")
 				}
 			}, "420s", "5s").Should(Succeed())
 
-			re := regexp.MustCompile("[0-9]+")
 			fmt.Printf("\nFetching pod status\n")
 			Eventually(func(g Gomega) {
-				pods, err := terraform.Pods(terraform.KubeConfigFile, false)
+				pods, err := terraform.Pods(KubeConfigFile, false)
+
 				g.Expect(err).NotTo(HaveOccurred())
+
+				re := regexp.MustCompile("[0-9]+")
```

Comment:
The `Eventually` block is basically a timeout loop, so this `re` variable will be redeclared on each loop iteration. We probably want it outside the loop in this case.